src/browserslist-config-canopy.test.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path')
+const browserslist = require('browserslist')
+const queries = require('./browserslist-config-canopy.js')
+
+describe('browserslist-config-canopy', () => {
+  it(`exports a valid browserslist config that is what we want for Canopy`, () => {
+    const browsers = browserslist(queries)
+    expect(browsers).toMatchSnapshot()
+  })
+})
```

Comment:
I can't comment on the serializer.js file below, since it doesn't have any code in it. Is it necessary?

Reply:
Oh good point -- I thought I was going to need a custom jest serializer but turns out I didn't